### ISSUE TITLE
USB camera and the use of v4l2-ctl --set-ctrl horizontal_flip=1

### ISSUE THREAD

**Issue by evandene** [motioneye-project/motioneye#2545]
Hardware and OS: Raspberry pi 2W Zero, latest Raspberry pi OS.

I'm in the process designing a inspection USB camera. When the camera rotates 180 degree the image should flip. In my setup I'm trying a terminal command "v4l2-ctl --set-ctrl horizontal_flip=1" but that doesn't work. The system comes with "unknown command" 'horizontal_flip'
Can somebody help this oldy solving the problem?
If I can get this to work, I would like to get this command under a tactile button.
![Horizontal_flip](https://user-images.githubusercontent.com/8578321/176104419-2048b2b0-8aa0-406e-ad67-7dacce4676ae.JPG)

**Comment by assetkad** [comment]
did you solve it?

**Comment by evandene** [comment]
Yes we did solve the problem by upgrading the firmware of the camera

**Comment by Marijn0** [comment]
There is also a Motion option to flip the camera image, but I would only use it if your camera does not support this feature natively:
https://motion-project.github.io/motion_config.html#flip_axis
